infrastructure/controller/pkg/endpoints/cluster_get.go

```diff
@@ -0,0 +1,96 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type ClusterGet struct {
+	clientSet *kubernetes.Clientset
+	namespace string
+}
+
+func (s *ClusterGet) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	configmapList, err := s.clientSet.CoreV1().ConfigMaps(s.namespace).List(r.Context(), v1.ListOptions{LabelSelector: "core.airy.co/component"})
+	if err != nil {
+		log.Printf("Unable to list config maps. Error: %s\n", err)
+		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
Do you think it'd be useful to return the error in a json response payload like we do with other endpoints?

Reply:
When it comes to internal server errors like kubectl errors. they should not be forwarded to the client